### DISCUSSION TITLE
Plans for the iOS client ?

### DISCUSSION THREAD

**Discussion by tobylibo** [stashapp/stash#5381] [Q&A]
Closed

**Comment by JanJastrow** [comment]
(Not a Dev here)
An iOS Client would be cool, but would probably not "fly" on the App Store.
You could develop one and either keep it on Apple Testflight indefinitely or (if you are an EU citizen) have it on the alternative AppStores (AltStore, etc).

In the meantime: The PWA (WebApp) works great 🚀

**Reply by taw123** [reply]
AppleTV client is the reason why a web app alone won't suffice, unfortunately.
